common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -534,12 +533,22 @@ def test_accordion_events(self):
                     'event': {
                         'target_name': 'Test Subsection 2,1 ',
                         'widget_placement': 'accordion',
+
                     }
                 },
             ],
-            outline_ui_events
+            selected_events
         )
 
+    def test_link_clicked_events(self):
+        self.course_nav.go_to_section('Test Section 1', 'Test Subsection 1,2')
+        self.course_nav.go_to_section('Test Section 2', 'Test Subsection 2,1')
+
+        filter_link_clicked = lambda event: event.get('name', '') == 'edx.ui.lms.link_clicked'
+        link_clicked_events = self.wait_for_events(event_filter=filter_link_clicked, timeout=2)
+
+        self.assertEqual(len(link_clicked_events), 2)
```

Comment:
The current URL also contains GUIDs, for what it's worth. Given that the functionality that generates the current and target url's is tested elsewhere, I don't think it's a problem not to test it here considering that the most we could test for is the beginning of each url.